packages/sanity/src/core/releases/components/BundleMenuButton/BundleMenuButton.tsx

```diff
@@ -51,10 +51,17 @@ export const BundleMenuButton = ({disabled, bundle, documentCount}: BundleMenuBu
       toast.push({
         closable: true,
         status: 'error',
-        title: 'Failed to delete bundle',
+        title: t('action.delete.failure'),
         description: e.message,
       })
     } finally {
+      toast.push({
+        closable: true,
+        status: 'success',
+        description: (
+          <Translate t={t} i18nKey={'action.delete.success'} values={{title: bundle.title}} />
+        ),
+      })
```

Comment:
This needs to be part of the `try` block, after the `await deleteBundle(bundle)` action

The `finally` block will be always executed, even if the action failed. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch
> The code in the finally block will always be executed before control flow exits the entire construct.